NEWS.md

```diff
@@ -0,0 +1,6 @@
+# codegrip (development version)
+
+* The long "flat" form for function definitions now aligns arguments using a
+  single indent rather than a double indent, which follows the
+  [tidyverse style guide](https://style.tidyverse.org/functions.html#long-lines-1)
```

Comment:
Well, itll reflect this after https://github.com/tidyverse/style/pull/223 is merged